### ISSUE TITLE
Liteloader extended from Forge with impact doesnt work minecraft 1.12.2

### ISSUE THREAD

**Issue by Wuvia** [ImpactDevelopment/ImpactIssues#1145]
**Describe the bug**
Tried using the Impact forge version for 1.12.2 with a version of forge that has a liteloader extension to it. It crashes without on startup. Any way to fix it ? Tried using the deleting the org folder and the urls. Doesn't work. Please help!

**Comment by 0-x-2-2** [comment]
Crash report?

**Comment by Wuvia** [comment]
Sorry had no time to post it here yesterday. But here you go :) 
https://pastebin.com/bKQLjPGZ

**Comment by ThebesAndSound** [comment]
I am having this issue too

**Comment by Wuvia** [comment]
Bump.

**Comment by User-developer** [comment]
Same

**Comment by leijurv** [comment]
`[11:30:58] [main/WARN]: Mixin config mixins.capi.json requires mixin subsystem version 0.7.8 but 0.7.5 was found. The mixin config will not be applied.` yikes

**Comment by LeafHacker** [comment]
Do we actually _need_ 0.7.8? We would probably be okay setting compatibility to 0.7.x?

**Comment by leijurv** [comment]
IIRC its something with maxShiftBy in injectors that requires 078

**Comment by n3rt46** [comment]
This is actually fairly simple to resolve. It would seem that you can actually just extract the version of Mixin that Impact downloads and then overwrite the version that LiteLoader uses at no consequence. 

In other words, if you already have the vanilla version installed, Impact will download Mixin 0.7.11-snapshot. Either convert it to a .zip and extract it, or open it using 7zip or similar. From there, do the same to the LiteLoader jar, and then just drag and drop in the Mixin files to the LiteLoader jar, overwriting it's older version of Mixin. After that, change the url for LiteLoader in its JSON, so that it doesn't redownload LiteLoader.

For sake of proving that this works (at least, on my machine), here's a download of the edit I made:
http://www.mediafire.com/file/19wjcub3c8ovasw/liteloader-1.12.2-SNAPSHOT.jar/file

MediaFire redundantly adds a .zip to the end of the .jar, so just remove that bit. i.e. .jar.zip -> .jar

**Comment by Orange65** [comment]
> This is actually fairly simple to resolve. It would seem that you can actually just extract the version of Mixin that Impact downloads and then overwrite the version that LiteLoader uses at no consequence.
> 
> In other words, if you already have the vanilla version installed, Impact will download Mixin 0.7.11-snapshot. Either convert it to a .zip and extract it, or open it using 7zip or similar. From there, do the same to the LiteLoader jar, and then just drag and drop in the Mixin files to the LiteLoader jar, overwriting it's older version of Mixin. After that, change the url for LiteLoader in its JSON, so that it doesn't redownload LiteLoader.
> 
> For sake of proving that this works (at least, on my machine), here's a download of the edit I made:
> http://www.mediafire.com/file/19wjcub3c8ovasw/liteloader-1.12.2-SNAPSHOT.jar/file
> 
> MediaFire redundantly adds a .zip to the end of the .jar, so just remove that bit. i.e. .jar.zip -> .jar

This did not work. I am using:
Forge1.12.2-14.23.5.2847
Liteloader 1.12.2 (latest version)
Impact forge 4.7.3
The game crashes before it loads.

**Comment by 0-x-2-2** [comment]
> EDIT 2: Using Impact forge 4.7.3 and WWE 2.7.2 for 1.12.2 at the same time causes the game to crash when ESC is pressed to bring up the pause menu. [Crash message](https://pastebin.com/b8samwWt). If anyone knows a solution it would be appreciated.

🤡. remove wwe....

**Comment by Orange65** [comment]
> > EDIT 2: Using Impact forge 4.7.3 and WWE 2.7.2 for 1.12.2 at the same time causes the game to crash when ESC is pressed to bring up the pause menu. [Crash message](https://pastebin.com/b8samwWt). If anyone knows a solution it would be appreciated.
> 
> 🤡. remove wwe....

I did you jackass

**Comment by 0-x-2-2** [comment]
"Using Impact forge 4.7.3 and WWE 2.7.2 for 1.12.2" ?

**Comment by Orange65** [comment]
> "Using Impact forge 4.7.3 and WWE 2.7.2 for 1.12.2"
> 
> ```
> WARNING: coremods are present:
>   MixinLoader (wwe.jar)
> ```
> 
> ?

I have temporarily removed WWE in the event someone else knows a solution.

**Comment by 0-x-2-2** [comment]
Does it still crash? If it does post an updated crash report.

**Comment by Orange65** [comment]
> Does it still crash? If it does post an updated crash report.

The game crashes if I run both impact and WWE at the same time.

As a temporary solution I have removed WWE so I can continue to play the game with just use impact on it's own. This makes the game function as intended.

In the event someone else knows how to stop the game crashing with both impact and WWE running, then I will continue to use both at the same time.

**Comment by 0-x-2-2** [comment]
Sounds like the issue is with WWE,

**Comment by leumasme** [comment]
This is still an issue.

**Comment by dominikaaaa** [comment]
Closing due to inactivity, and I doubt anybody actually uses liteloader in 2020

**Comment by BBloom81** [comment]
> Closing due to inactivity, and I doubt anybody actually uses liteloader in 2020

Right, because no one plays on 2b2t and needs liteloader for world downloader /s

**Comment by l1ving** [comment]
World downloader is a forge mod though???

**Comment by leumasme** [comment]
> World downloader is a forge mod though???

Not sure where you found that, but i can only find 1.14 - 1.16 forge versions of WDL.
Impact only has 1.12.2 forge support.

**Comment by l1ving** [comment]
WDL 1.12.2 for Forge. It's on the official WDL page (https://github.com/pokechu22/worlddownloader/releases)
[wdl.zip](https://github.com/ImpactDevelopment/ImpactIssues/files/5386591/wdl.zip)

**Comment by leumasme** [comment]
![image](https://user-images.githubusercontent.com/29953391/96162883-618a8880-0f19-11eb-9f0c-36fd9f661c37.png)
Am i blind?

**Comment by l1ving** [comment]
You're not looking at the right release. I built the jar myself off of 4.0.1.7, and I already [**linked**](https://github.com/ImpactDevelopment/ImpactIssues/files/5386591/wdl.zip) it to you.

**Comment by BBloom81** [comment]
Thanks but no thanks.  That version is missing several key features over the liteloader version, making it unusable imo.

**Comment by l1ving** [comment]
It's literally identical unless you're using a different version on liteloader. Build the latest jar for forge 1.12.2 yourself, 4.0.1.7 isn't latest

**Comment by BBloom81** [comment]
Why would I not use the latest liteloader version for 1.12.2?  Of course I am.  They specifically state as well that Forge support is barely a thing.
I'm sorry but you've yet to fully read a comment from me, nor the issue I posted asking for help.
You're read a word or two and made assumptions.  I can not spend my day, repeating myself in different ways.

I'd say that if anyone actually can help people get Forge+Liteloader working with Impact for 1.12.2, that'd be nice.  As for me, I don't care anymore.  As I told you in my issue post, I was gonna just buy Future and I did.  I have zero time for this nonsense.
